lib/perl/lib/scripts/extract_from_pdb

```diff
@@ -1236,12 +1236,12 @@ sub get_unrealeased_file
 	if (-e $cache_file){return $cache_file;}
 	elsif ( $env_file && ! -e $env_file)
 	  {
-	    &url2file("http://www.rcsb.org/pdb/rest/getUnreleased",$env_file);
+	    &url2file("https://ftp.rcsb.org/pub/pdb/holdings/unreleased_entries.json.gz",$env_file);
 	    if ( -e $env_file && !-z $env_file){return $env_file;}
 	  }
 	else
 	  {
-	    add_warning($$,$$,"UNREALEASED_FILE must be set to the location of your unrealeased.xml file as downloaded from http://www.rcsb.org/pdb/rest/getUnreleased when using NO_REMOTE_PDB_DIR=1");
+	    add_warning($$,$$,"UNREALEASED_FILE must be set to the location of your unrealeased.xml file as downloaded from https://ftp.rcsb.org/pub/pdb/holdings/unreleased_entries.json.gz when using NO_REMOTE_PDB_DIR=1");
```

Comment:
@athbaltzis, maybe here changed the warning message to mention the new `json` format?